.github/workflows/tag-and-publish.yml

```diff
@@ -71,33 +71,6 @@ jobs:
       - name: Push GCR image
```

Comment:
I have left publishing of new image purposefully since there is no harm in having it here and only removed the updating helm charts portion from the Github workflow.

Reply:
and this whole repo is going away soon, anyway!